tools/leaflet/leaflet_amd_loader_config.js

```diff
@@ -0,0 +1,59 @@
+(function() {
```

Comment:
Why is this necessary?

Reply:
Otherwise leaflet can not be found when running tests. Is there another possible solution for that?

Reply:
I found a solution, sorry. I forgot to add the dependency correctly. This is not necessary indeed.